pkg/asset/installconfig/gcp/client.go

```diff
@@ -222,6 +224,34 @@ func (c *Client) GetProjects(ctx context.Context) (map[string]string, error) {
 	return projects, nil
 }
 
+// GetProjectRegions gets the regions that are valid for the project. An error is returned when unsuccessful
+func (c *Client) GetProjectRegions(ctx context.Context, project string) ([]string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
This context is shared between two calls. I think it should probably be reset for each.